trino-connector/trino-connector/src/main/java/org/apache/gravitino/trino/connector/GravitinoConfig.java

```diff
@@ -29,25 +29,37 @@
 import java.util.Properties;
 import org.apache.commons.lang3.StringUtils;
 
+/** Gravitino config. */
 public class GravitinoConfig {
 
   // Trino config keys
+  /** The Trino discovery URI. */
   private static final String TRINO_DISCOVERY_URI = "discovery.uri";
+  /** The Trino catalog config directory. */
   private static final String TRINO_CATALOG_CONFIG_DIR = "catalog.config-dir";
+  /** The Trino plugin bundles. */
   public static final String TRINO_PLUGIN_BUNDLES = "plugin.bundles";
+  /** The Trino catalog store. */
   public static final String TRINO_CATALOG_STORE = "catalog.store";
+  /** The Trino catalog management. */
   public static final String TRINO_CATALOG_MANAGEMENT = "catalog.management";
```

Comment:
The above comments are not really needed. There is little point in adding a comment that merely reproduces what the code states.

Reply:
However, if they are required by Javadocs to remove warnings, then that's okay - as I mentioned in my other comments below.